models/daos/transactions/gnosis/daos_transactions_gnosis_schema.yml

```diff
@@ -0,0 +1,114 @@
+version: 2
+
+models:
+  - name: daos_transactions_gnosis_erc20 
+    meta:
+      blockchain: gnosis
+      sector: daos
+      contributors: henrystats 
+    config:
+      tags: [ 'gnosis','erc20','daos', 'dao creator tools', 'henrystats' ]
+    description: >
+      erc20 transactions of DAO wallets 
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date 
+            - blockchain
+            - dao_creator_tool
+            - dao
+            - dao_wallet_address
+            - tx_hash 
+            - tx_index
+            - tx_type
+            - trace_address
+            - address_interacted_with
+            - value 
+            - asset 
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "Blockchain which transaction happened"
+      - &dao_creator_tool
+        name: dao_creator_tool
+        description: "Name of creator tool used to create DAO"
+      - &dao
+        name: dao
+        description: "address of the dao deployed"
+      - &dao_wallet_address
+        name: dao_wallet_address
+        desciption: "wallet address used by DAO to receive & transfer funds"
+      - &block_date
+        name: block_date
+        description: "UTC event block date of tramsaction"
+      - &block_time
+        name: block_time
+        description: "UTC event block time of transaction"
+      - &tx_type
+        name: tx_type
+        description: "transaction type (deposit or withdrawal)"
+      - &asset
+        name: asset
+        description: "token or asset transferred or received"
+      - &raw_value
+        name: raw_value
+        description: "raw value of asset/token transferred"
+      - &value
+        name: value 
+        description: "converted value of asset/token transferred"
+      - &usd_value
+        name: usd_value
```

Comment:
Do you want to test the gnosis model as well?

Reply:
there's currently an issue with gnosis data where there are some missing transactions, I could run test queries (I already have them set up and included in the code) and then copy the info from gnosis scan. 

But I can do that within the next half an hour or so so yeah let me get onto it.

Reply:
issue on dune*

Reply:
so just checked and I don't have tests for two dao models on gnosis (a transaction query), will have to check the tx_hashes if they pop up.